starboard/nplb/posix_compliance/posix_file_temp_test.cc

```diff
@@ -34,28 +34,37 @@ TEST_F(FileTempTest, MkdtempBasicSuccess) {
   char buffer[tmpl.length() + 1];
   strcpy(buffer, tmpl.c_str());
 
-  char* result = mkdtemp(buffer);
-  ASSERT_NE(result, nullptr);
+  char* const result = mkdtemp(buffer);
+  ASSERT_NE(result, nullptr)
```

Comment:
Pro-tip: `ASSERT_THAT(result, ::testing::NotNull()));`, here and
below.

https://google.github.io/googletest/reference/matchers.html#generic-comparison